webserver/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "webserver"
```

Comment:
I suggest changing to "bughouse-webserver". When you `use` crates from other crates it's not immediately obvious whether they are local or come from `crates.io`, so I've decided to prefix all crates in this package with “bughouse” to remove this uncertainty.

Reply:
Done.